posydon/interpolation/IF_interpolation.py

```diff
@@ -598,8 +602,8 @@ def _setValid(self, ic, X):
         which = np.isnan(np.sum(X, axis=1))
         valid[which] = -1
         print(f"Discarded {np.sum(which)} binaries with nans in input values.")
-
-        for i, flag in enumerate(self.interp_classes):
+        valid_classes = ['stable_MT', 'unstable_MT', 'no_MT', 'stable_reverse_MT']
```

Comment:
Again, this hard codes stuff and takes flexibility away. @philipp-rajah, please check that this is what we want.

Reply:
Wouldn't it be better to add a parameter to the interpolation class, which contains the `valid_classes`?
I'm not sure how others think about, but I'd may say it would be better to have a negative version, hence which of the interpolation classes to exclude from validation. Opinions?